src/ResourceManager/Automation/Commands.Automation/Common/AutomationPSClientDSC.cs

```diff
@@ -34,6 +37,7 @@
 using Job = Microsoft.Azure.Management.Automation.Models.Job;
 using JobSchedule = Microsoft.Azure.Management.Automation.Models.JobSchedule;
 using Schedule = Microsoft.Azure.Commands.Automation.Model.Schedule;
+using Microsoft.Azure.Management.ResourceManager;
```

Comment:
Keep in mind you won't be able to use this when merging back to our main branch.

Reply:
sure, my understanding is that in the main branch all would be on latest version. as we saw that Profile module is dependent on the older version of resources module

Reply:
Modules on the main branch use our [internal version of the ResourceManager SDK](https://github.com/Azure/azure-powershell-common/tree/develop/src/ResourceManager). A new version of this SDK would need to be created in our Common repo. Then, your changes would need to use that version. This is not necessary for a preview module, but will not be allowed in our main repo.